helix-core/src/main/java/org/apache/helix/ConfigAccessor.java

```diff
@@ -604,6 +605,61 @@ public RESTConfig getRESTConfig(String clusterName) {
     return new RESTConfig(record);
   }
 
+  /**
```

Comment:
If there is a change to config accessor, should we add tests in TestConfigAccessor as well?

Reply:
There are tests for rest config covering set and update in TestConfigAccessor. I just added a few more cases for delete.